testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/OutputScrapingExecutionResult.java

```diff
@@ -40,12 +40,30 @@
 import java.util.regex.Pattern;
 
 public class OutputScrapingExecutionResult implements ExecutionResult {
-    // This monster is to find lines in our logs that look like stack traces
-    // We want to match lines that contain just packages and classes:
-    // at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$1(ExecuteActionsTaskExecuter.java:145)
-    // and with module names:
-    // at java.base/java.lang.Thread.dumpStack(Thread.java:1383)
-    static final Pattern STACK_TRACE_ELEMENT = Pattern.compile("\\s+(at\\s+)?([\\w.$_]+/)?[a-zA-Z_][\\w.$]+\\.[\\w$_ =+'-<>]+\\(.+?\\)(\\x1B\\[0K)?");
+    /*
+     This monster is to find lines in our logs that look like stack traces
+
+     We want to match lines that contain just packages and classes:
+        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$1(ExecuteActionsTaskExecuter.java:145)
+
+     and with module names:
```

Comment:
💅 Modules are more complex than this and could also contain a classloader name and module version, see https://stackoverflow.com/q/77399235/8006558

An example stacktrace is in: https://github.com/gradle/gradle/issues/6726, I've also seen cases where platform modules like java.base had a version attached to them but have a hard time finding an example